docs/pages/installation/index.md

```diff
@@ -3,18 +3,23 @@ layout: page
 title: Installation Guide
 permalink: /installation
 ---
+
 <a name="getting-started"></a>
 
-# Quick Start 
+# Quick Start
+
 Getting Meshery up and running on a locally on Docker-enabled system is easy. Use the Meshery command line interface, `mesheryctl`, to start Meshery on any of its [supported platforms](/docs/installation/platforms).
 
 ## Using `mesheryctl`
+
```

Comment:
@iknowahra, thank you for improving the overall formatting of this markdown. Nice! 👍